habitat-lab/habitat/gym/gym_wrapper.py

```diff
@@ -178,12 +175,18 @@ class HabGymWrapper(gym.Env):
     Example usage:
     """
 
-    def __init__(self, env, save_orig_obs: bool = False):
-        gym_config = env.config.gym
-        self._gym_goal_keys = gym_config.desired_goal_keys
-        self._gym_achieved_goal_keys = gym_config.achieved_goal_keys
-        self._gym_action_keys = gym_config.action_keys
-        self._gym_obs_keys = gym_config.obs_keys
+    def __init__(
+        self,
+        env: "RLEnv",
+        habitat_gym_config: "DictConfig",
```

Comment:
What is the reason for adding this new argument where it did not seem needed from the current code?

Reply:
Removed. Use `env .config.gym` instead.